components/button/test/button-icon.visual-diff.html

```diff
@@ -13,21 +13,11 @@
 	<meta charset="UTF-8">
 	<style>
 		.translucent-container {
-			background-color: orange;
-			position: relative;
 			width: 200px;
 			height: 80px;
 		}
-		.translucent-container > img {
-			border-radius: 6px;
-			height: 100%;
-			position: absolute;
-			width: 100%;
-		}
```

Comment:
Couldn't figure out what this was trying to do -- removed.

Reply:
🎉 .  I think that was left-over from when the visual-diff fixtures was set up like the demo, to include an actual image in the background... to look all fancy and such.  It was removed because the async nature of the image load make the test flakey, slower, and basically added little value for the test.